backend/fms_core/serializers.py

```diff
@@ -268,3 +271,20 @@ class Meta:
         model = Group
         fields = ("id", "name", "permissions")
         depth = 1
+
+class ProjectSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Project
+        fields = "__all__"
+
+class ProjectExportSerializer(serializers.ModelSerializer):
+    created_by = serializers.CharField(read_only=True, source="created_by.username")
```

Comment:
You can remove the tracking fields from the export serializer (created_by, updated_by, created_at, updated_at). I don't think this is information that someone would want exported.

Reply:
Perfect!